app/models/contact_info.rb

```diff
@@ -12,7 +12,7 @@ class ContactInfo < ActiveRecord::Base
   validates :type, presence: true, inclusion: { in: VALID_TYPES }
   validates :value,
             presence: true,
-            uniqueness: {scope: [:type]}
+            uniqueness: {scope: [:user_id, :type]}
```

Comment:
They can according to our database schema...

Are you saying that they shouldn't? :)